source/ui/ui-updater.cpp

```diff
@@ -161,11 +161,12 @@ streamfx::ui::updater::updater(QMenu* menu)
 
 	{ // Retrieve the updater object and listen to it.
 		_updater = streamfx::updater::instance();
-		_updater->events.automation_changed.add(std::bind(&streamfx::ui::updater::on_updater_automation_changed, this,
+		auto& events = _updater->events;
+		events.automation_changed.add(std::bind(&streamfx::ui::updater::on_updater_automation_changed, this,
 														  std::placeholders::_1, std::placeholders::_2));
-		_updater->events.channel_changed.add(std::bind(&streamfx::ui::updater::on_updater_channel_changed, this,
+		events.channel_changed.add(std::bind(&streamfx::ui::updater::on_updater_channel_changed, this,
 													   std::placeholders::_1, std::placeholders::_2));
-		_updater->events.refreshed.add(
+		events.refreshed.add(
```

Comment:
**Code Clarity:** This reduces code clarity with no noticable gain. Compilers already do this optimization on `-O1` (or with a single folding iteration).